templates/single.twig

```diff
@@ -39,7 +39,11 @@
 						<div class="single-post-meta">
 							<address class="single-post-author">
 								{{ __( 'by', 'planet4-master-theme' ) }}
-								<a href="{{ post.author.path }}">{{ post.author.name }}</a>
+								{% if not ( post.get_author_override ) %}
```

Comment:
Just a notice here as well to be sure you saw this:
P4_Post and P4_User objects are (after 1.24.0) more complete and P4_Post has since its creation a P4_User in it which actually is a Timber/User. We can deprecate `get_author_override` and `get_custom_terms` and use `post.author` and `post.page_types` directly in the templates having the full data provided by Timber for Timber/Post and Timber/User objects.
#629 
@sagarsdeshmukh @kirdia @pablocubico